components/raftstore/src/coprocessor/dispatcher.rs

```diff
@@ -780,6 +788,17 @@ impl<E: KvEngine> CoprocessorHost<E> {
         true
     }
 
+    /// Ignore the message if one of the observers returns true.
+    pub fn on_raft_message(&self, msg: &RaftMessage) -> bool {
+        for observer in &self.registry.message_observers {
+            let observer = observer.observer.inner();
+            if observer.on_raft_message(msg) {
+                return true;
```

Comment:
It is conflict with all other APIs that they returns false to represent the request/message should be dropped.

Reply:
Returning false means not drop while returning true means do drop. The message will be dropped if there are at least one observers have choosen to drop. Should I use a more explicit enum instead of bool to make the return value more clear?

Reply:
You should follow the style how other APIs are defined.

Reply:
Do you mean pass a `should_skip: &mut bool` rather than making the observer return a bool?

Reply:
I mean "returns false to represent the request/message should be dropped".